### ISSUE TITLE
[Adapter Cards-Schema Explorer]: Tool tip is missing for Date and Time icons present under the adaptive cards.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7869]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Screen Reader: Narrator
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Repro Steps:
1.Open above URL with V- credential.
2.Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
3.Navigate to 'Input.Date" link present under Input from the left navigation using the tab key and activate it using Enter key.
4. Navigate to the adaptive cards present under it using the Tab key.
5. Navigate to the "Date Input" heading under Adaptive Card using Tab key.
6. Navigate using Tab key to the Date Picker.
7. Observe the issue here.

Actual Result:
Tool tip is not present on the Date Picker icon.

Expected Result:
Tool tip should be given for the date picker as "Calender".  

Observation:
Same issue exists for the Time picker icon present under the "Input.Time" link under the input.
Navigate to the "What time do you want to meet" heading under Adaptive Card using Tab key.

User Experience:
Keyboard users and cognitive users will get impacted if Tool tip is not given for the Date and Time Icons.

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.03

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

<img width="960" alt="7 there is no tooltip for clock and calender" src="https://user-images.githubusercontent.com/104482612/190393927-beb3038e-fa6a-419c-81b6-9fdc1d5a61ac.png">
<img width="960" alt="COde snippet for date" src="https://user-images.githubusercontent.com/104482612/190396948-73348820-4f29-4f45-b758-8d1f32b68bf1.png">
<img width="960" alt="COde snippet for time" src="https://user-images.githubusercontent.com/104482612/190396977-98a08096-182c-4812-b079-b25f617ec2f1.png">

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.
